sql/2019/05_Third_Parties/05_09.sql

```diff
@@ -0,0 +1,19 @@
+#standardSQL
+# Top 100 third party requests by request volume
+SELECT
+  requestUrl,
+  COUNT(*) AS totalRequests,
```

Comment:
You may also want to add a pctRequests field. Out of context `5662539` could either be a very large or very small number, relatively speaking.

Reply:
BigQuery wasn't able to execute the over query and failed with a memory limit exceeded on this one, just make a separate SELECT in the from for the overall count then?